handlers/handlers.go

```diff
@@ -21,12 +27,67 @@ var reCaptcha = recaptcha.R{
 	Secret: os.Getenv("RECAPTCHA_SECRET_KEY"),
 }
 
+//the function for the minifying
+var m = minify.New()
+
+func compileTemplates(templatePaths ...string) (*template.Template, error) {
+
+	var tmpl *template.Template
+	for _, templatePath := range templatePaths {
+		name := filepath.Base(templatePath)
+		if tmpl == nil {
+			tmpl = template.New(name)
+		} else {
+			tmpl = tmpl.New(name)
+		}
+
+		b, err := ioutil.ReadFile(templatePath)
+		if err != nil {
+			return nil, err
+		}
+
+		mb, err := m.Bytes("text/html", b)
+		if err != nil {
+			return nil, err
+		}
+		tmpl.Parse(string(mb))
+	}
+	return tmpl, nil
+}
+
+func minifyCSSFiles(templateDirectory string) {
+	cssFileDirectory := filepath.Join(templateDirectory, "../assets/css/")
+	cssFilePaths, _ := filepath.Glob(filepath.Join(cssFileDirectory, "*.css"))
+	for _, cssFilePath := range cssFilePaths {
+		if cssFilePath[len(cssFilePath)-8:] != ".min.css" {
```

Comment:
Instead of 

```
for _, value := range values {
    if value.hasProperty() {
        // Conditional code ...
    }
}
```

To [reduce nesting](http://www.refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html), consider using 

```
for _, value := range values {
    if !value.hasProperty() {
        continue
    }

    // Conditional code ...
}
```